Library/SharedFunctions.swift

```diff
@@ -779,15 +777,8 @@ private func estimatedMinMax(
       return
     }
 
-    /// Only update min if the current rewards esitmatedMin is smaller.
-    if min == 0 || estimatedMin < min {
-      min = estimatedMin
-    }
-
-    /// Only update max if the current rewards esitmatedMax is bigger.
-    if max == 0 || estimatedMax > max {
-      max = estimatedMax
-    }
+    min = min == 0 ? estimatedMin : min + estimatedMin
+    max = max == 0 ? estimatedMax : max + estimatedMax
```

Comment:
Nit: Do we need to check for `0` here? Why not just `min += estimatedMin` and `max += estimatedMax`? In the case where ex `min == 0`, `min + estimatedMin` and `estimatedMin` have the same value. I could be missing something, but I find `min += estimatedMin` to be more common/predicable and thus quicker to understand.